QUICKSTART.md

```diff
@@ -125,6 +128,77 @@ asymmetry of the KNL NUMA configuration the '-S'/'--pes-per-numa-node'
 is not appropriate on the Theta system, but this option is also not
 interpreted by the geopmaprun wrapper.
 
+### Failure of msr-safe
+
+As described in the [Run Requirements : MSR
+Driver](../../../../geopm/geopm#msr-driver) section of the main GEOPM readme,
```

Comment:
I'm dumbfounded that this ".." notation works to switch between github repos.  Would it maybe be better to give a full URL link?